pod-operation/hx_wheel_encodeer

```diff
@@ -0,0 +1,39 @@
+use std::thread::sleep;
+use std::time::Duration;
+use rppal::gpio::{Gpio, Level};
+
+const OUTPUT_A: u8 = 6;
+const OUTPUT_B: u8 = 7;
+
+fn main() {
+    let mut counter = 0;
+    let mut a_state;
+    let mut a_last_state;
```

Comment:
Partly my fault for omitting details on #11, but similar to the other sensor implementations, we'll want an abstraction for the wheel encoder as a struct which maintains its own fields. With only a testing script like this, we won't be able to incorporate the measurements into the FSM. Particularly, for running demos with loops, we have a separate `demo.rs` file where we can define testing functions running instances of the abstraction.

Reply:
You can also refer to [`pressure_transducer.rs`](https://github.com/UCI-HyperXite/hx9-control-system/blob/main/pod-operation/src/components/pressure_transducer.rs) as an example on making the abstraction.

Reply:
Thanks for pointing it out! I’ll resolve this by EOW